test/test_transformers.py

```diff
@@ -849,6 +849,42 @@ def test_encoder_is_causal(self):
 
         self.assertEqual(masked_output, is_causal_output)
 
+    @onlyCUDA
+    def test_math_backend_high_precision(self):
+        xq = torch.rand([1, 6432, 2, 80], device="cuda", dtype=torch.bfloat16) * 5
+        xk = torch.rand([1, 6432, 2, 80], device="cuda", dtype=torch.bfloat16) * 5
+        xv = torch.randn([1, 6432, 2, 80], device="cuda", dtype=torch.bfloat16)
```

Comment:
can you get away with smaller sizes during testing?

Reply:
Changed 6432 to 128 and we can capture the issue with the test.